pkg/restore/restore.go

```diff
@@ -949,6 +959,7 @@ func (ctx *restoreContext) restoreItem(obj *unstructured.Unstructured, groupReso
 			return warnings, errs
 
 		case hasDeleteReclaimPolicy(obj.Object):
+			fmt.Printf("Has a delete policy!")
```

Comment:
probably want to remove this

Reply:
Though I had gotten all my debug prints, thanks :)